### ISSUE TITLE
Error 500 when quering collections

### ISSUE THREAD

**Issue by 0xbase12** [nuvla/api-server#179]
The same stack was also visible for credential and event collection
```
nuvla_api.1.ui5watfg3zuh@linuxkit-025000000001    |  java.lang.UnsupportedOperationException: nth not supported on this type: PersistentArrayMap
nuvla_api.1.ui5watfg3zuh@linuxkit-025000000001    |               RT.java:987 clojure.lang.RT.nthFrom
nuvla_api.1.ui5watfg3zuh@linuxkit-025000000001    |               RT.java:936 clojure.lang.RT.nth
nuvla_api.1.ui5watfg3zuh@linuxkit-025000000001    |          std_crud.clj:109 sixsq.nuvla.server.resources.common.std-crud/query-fn[fn]
nuvla_api.1.ui5watfg3zuh@linuxkit-025000000001    | infrastructure_service.clj:183 sixsq.nuvla.server.resources.infrastructure-service/eval31221[fn]
nuvla_api.1.ui5watfg3zuh@linuxkit-025000000001    |          MultiFn.java:229 clojure.lang.MultiFn.invoke
nuvla_api.1.ui5watfg3zuh@linuxkit-025000000001    |             routes.clj:17 sixsq.nuvla.server.app.routes/fn[fn]
nuvla_api.1.ui5watfg3zuh@linuxkit-025000000001    |              core.clj:158 compojure.core/wrap-response[fn]
nuvla_api.1.ui5watfg3zuh@linuxkit-025000000001    |              core.clj:128 compojure.core/wrap-route-middleware[fn]
nuvla_api.1.ui5watfg3zuh@linuxkit-025000000001    |              core.clj:137 compojure.core/wrap-route-info[fn]
nuvla_api.1.ui5watfg3zuh@linuxkit-025000000001    |              core.clj:146 compojure.core/wrap-route-matches[fn]
nuvla_api.1.ui5watfg3zuh@linuxkit-025000000001    |              core.clj:185 compojure.core/routing[fn]
nuvla_api.1.ui5watfg3zuh@linuxkit-025000000001    |             core.clj:2701 clojure.core/some
...
```

**Comment by 0xbase12** [comment]
Request:
`curl 'https://localhost/api/infrastructure-service' -X PUT -H 'accept: application/json' -H 'Referer: http://localhost:8280/ui/data' -H 'Origin: http://localhost:8280' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.157 Safari/537.36' -H 'content-type: application/x-www-form-urlencoded' --data $'filter=(id%3D\'infrastructure-service%2F27b90f73-7f26-4283-8436-e8648ae3a172\')&select=id%2C%20name%2C%20description%2C%20subtype' --compressed --insecure`

Response:
```
{
  "status" : 500,
  "message" : "Unexpected exception thrown: nth not supported on this type: PersistentArrayMap"
}
```

**Comment by 0xbase12** [comment]
Fixed in following commit ff8fb853
